pom.xml

```diff
@@ -224,7 +231,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                	<artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>-Xms512m -Xmx2048m -XX:MaxPermSize=768m -XX:ReservedCodeCacheSize=128m</argLine>
+                    <systemPropertyVariables>
+                        <org.jboss.logging.provider>jdk</org.jboss.logging.provider>
```

Comment:
Could you perhaps share your JDK logging config - if you have any? Maybe it's me doing it wrong.